boards/xtensa/esp32s3_devkitm/Kconfig.board

```diff
@@ -6,6 +6,10 @@ config BOARD_ESP32S3_DEVKITM
 	bool "ESP32S3 DevKitM Board"
 	depends on SOC_SERIES_ESP32S3
 
+config BOARD_ESP32S3_APPCPU
+	bool "ESP32S3 Board configuration for APPCPU (core 1)."
+	depends on SOC_SERIES_ESP32S3 && SOC_ESP32S3_APPCPU
+
```

Comment:
nit: I found this in the first commit with the SoC changes, but I think this belongs in the second commit where the board is added.